examples/rust/components/http-keyvalue-watcher/src/lib.rs

```diff
@@ -0,0 +1,135 @@
+use crate::bindings::wasi::http::outgoing_handler;
+use crate::bindings::wasi::http::types::*;
+use wasmcloud_component::wasi::logging::logging::{log, Level};
+mod bindings {
+    use super::KvWatcher;
```

Comment:
Nit: Is there a reason this and the `export!` are here? They look a little weird here and you can just do `bindings::export!(KvWatcher);` at the top level